src/client/src/ui/spotTile/components/notifications/NotificationContainer.tsx

```diff
@@ -34,7 +34,7 @@ export default class NotificationContainer extends PureComponent<Props> {
       )
     }
     if (!lastTradeExecutionStatus || !hasNotification(lastTradeExecutionStatus)) {
-      return () => null as SFC<void>
+      return (() => null) as SFC<object>
```

Comment:
I thought about this. We should type the functions return type instead of doing this.